src/controllers/notifications.controller.js

```diff
@@ -31,6 +31,16 @@ const NotificationController = {
       'Notification status updated!'
     );
   },
+
+  async markAllNotificationsAsRead(req, res) {
+    const { userId } = req.userData;
+    await NotificationService.markAllAsRead(userId);
```

Comment:
Good job @Crispy-lang  for this implementation. However,  even if the user doesn't have any notification will receive a descriptive message that ' All notifications are marked as read'. can you please fix it?

Reply:
No, @alexismajya this will be handled in the front-end where they will be a button and that button will be disabled whenever they are no notifications to read